notes/CodeFlag_notes.csv

```diff
@@ -138,3 +138,5 @@ noteID,note
 141,"See Appendix XYZ."
 142,"In the ring ordering scheme, the 12 rhomboids are not considered and the scanning mode applies to the grid in its entirety."
 143,"In the nested ordering scheme, the 12 rhomboids are scanned in order defined in Gorsky et al. 2015. The flag bits then refer to how the points are scanned within a rhomboid."
+221a,"For code number 102 the drought index encoded in the message is specified by the table entries from discipline 0 and parameter category 22 with the corresponding parameter number. The statistical processing period of the drought index is defined by the keys ""length of time range"" and ""indicator of unit of time for the increment"""
+221b,"Descriptions of these indices are available in the Handbook of Drought Indicators and Indices (WMO-No. 1173) [https://library.wmo.int/idurl/4/55169]. The indices are specified by setting discipline and parameter category/number to the specific index to be encoded. Drought indicators are calculated over different time ranges, prescribed by the 'Length of time range' and 'Indicator of unit for time range' parameters in the templates 4.107, 4.108, 4.109 and 4.112. All standardized drought indices are calculated according to the Standardized Precipitation Index User Guide (WMO-No. 1090)[https://library.wmo.int/idurl/4/39629]."
```

Comment:
@sebvi please review revised note.

Reply:
for reference: https://github.com/wmo-im/GRIB2/pull/237/commits/84db17974b7dfa0c393adc17bd3afc926c10accc